tools/merge-databases/merge-databases.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+# The purpose of this script is a one time data migration from
+# builder_originsrv and builder_sessionsrv -> builder
```

Comment:
also jobsrv

we'll need this to also handle on-prem (no jobsrv) or create a different variant for on-prem. 

could you also open an issue to follow up on the on-prem work after this merges..